aurelia-slickgrid/src/aurelia-slickgrid/slick-pagination.ts

```diff
@@ -171,6 +171,17 @@ export class SlickPaginationCustomElement {
     } else {
       throw new Error('Pagination with a backend service requires "BackendServiceApi" to be defined in your grid options');
     }
+
+    // dispatch the changes to the parent component
+    this.elm.dispatchEvent(new CustomEvent(`on-pagination-changed`, {
```

Comment:
I didn't want to use the `sg` prefix because it represents the SlickGrid events not Aurelia-Slickgrid events. I didn't feel the need to have a prefix for internal one. Do you really think it's necessary?

Reply:
I prefixed the events with `sg` to uniquely identify any event coming from your library that is exposed to the user (i see this in many aurelia libraries). Since these events are bubbling from your library they will always be exposed to the user. In my application I can bind to that `on-pagination-changed` event and any other event, and if we prefixed some events and not others that are exposed I thought that may come across as inconsistent. For example:

```html
<template>
  <aurelia-slickgrid 
    on-pagination-changed="handlePagination()"
    sg-on-cell-click="handleClick()">
  </aurelia-slickgrid>
</template>
```

Reply:
hmm I would still rather make a distinction between my own events versus the SlickGrid events which always have an `event` and `args` and are not working the same as internal ones.

Reply:
Okay I understand. You may still want a different prefix because another library can have the same event name `on-pagination-changed` that would be in conflict with yours since they all bubble up the DOM

Reply:
ok perhaps `as` (Aurelia-Slickgrid), hmm no that's too confusing lol... 
or `asg` that's probably better